isaac_msgs/action/InspectionV2.action

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) 2017, United States Government, as represented by the
```

Comment:
The difference between V1 and V2 is only added fields, so no data loss if we make a bmr rule from V1 to V2 instead.
Please note that bmr rules are preferred from a SW quality point of view, no point in having extra legacy message definitions being compiled.

Reply:
I think you're saying that you want any `InspectionV1` messages bmr migrated to `InspectionV2` so the `InspectionV1` action definition is not needed.

Unfortunately, in order to fix a bag that contains legacy `Inspection` messages that really conform to the `InspectionV1` type rather than the current `Inspection` definition, we do need the frozen `InspectionV1` message definition so we have a target for the rewrite rule.

You're correct that then we could add a bmr rule to migrate `InspectionV1` messages to `InspectionV2`, but that wouldn't get rid of the necessity to define `InspectionV1`. If you'd like to add such a bmr rule, maybe in a new PR, I would support that.

Reply:
The above may make more sense if I note that `rosbag_fix_all.py` invokes all rewrite rules first then all bmr rules. It does it in that order because the `rosbag fix` command that invokes bmr rules is very picky and won't be happy if it can't finish migrating everything.

So as the script is currently written, you can't apply a bmr rule followed by a rewrite rule. And it might not be easy to relax that restriction.